package.json

```diff
@@ -18,8 +18,8 @@
   },
   "repository": "https://github.com/mirego/ember-boilerplate",
   "engines": {
-    "node": "10.15.1",
-    "npm": "6.4.1"
+    "node": "~> 10.15",
```

Comment:
I don’t think `~>` is supported as a valid version range by npm, this should be `~10.15` or `10.15.x` if we follow what’s specified in the docs [here](https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1) and [here](https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-)

Reply:
But it is 😄 

```js
const semver = require('semver');

semver.satisfies('10.14.5', '~10.14')
// true

semver.satisfies('10.14.5', '~> 10.14')
// true
```

Since [both are allowed](https://github.com/npm/node-semver/blob/master/semver.js#L1001-L1011), I think we should keep using `~>` since it’s also used in Elixir and Ruby (while `~` is not).

What do you think?

Reply:
> I think we should keep using ~> since it’s also used in Elixir and Ruby

I know we want to bring uniformity to all our boilerplates but I think we should follow the stack’s community standards instead of cross-stacks standards. I’ve rarely seen `~>` used in node land and while it’s supported it’s not documented… I don’t know, I can live with it but this still feels weird 😄

Reply:
No you’re right, we should follow NPM’s community standards. I’ll make the change in all of our boilerplates `package.json` files 😄